zerver/lib/actions.py

```diff
@@ -557,7 +557,7 @@ def do_create_user(email: str, password: Optional[str], realm: Realm, full_name:
                    default_stream_groups: Sequence[DefaultStreamGroup]=[],
                    source_profile: Optional[UserProfile]=None,
                    realm_creation: bool=False,
-                   acting_user: Optional[UserProfile]=None) -> UserProfile:
+                   *, acting_user: Optional[UserProfile]) -> UserProfile:
```

Comment:
Can you rebase off the latest master?  In https://github.com/zulip/zulip/commit/d0ba3cadcfb6cf6c58b99dd87a00f489fd438c6e I clean up the formatting of `do_create_user`.

You'll get a conflict here, and I believe you'll just want to accept my changes and then add "*," on its own line.